Index: talk/app/webrtc/java/jni/peerconnection_jni.cc
===================================================================
--- talk/app/webrtc/java/jni/peerconnection_jni.cc	(revision 4804)
+++ talk/app/webrtc/java/jni/peerconnection_jni.cc	(working copy)
@@ -56,6 +56,7 @@
 #undef JNIEXPORT
 #define JNIEXPORT __attribute__((visibility("default")))
 
+#include <unistd.h>
 #include <asm/unistd.h>
 #include <limits>
 #include <map>
